src/components/ItemTable/ItemTable.tsx

```diff
@@ -28,18 +28,26 @@ const ItemTable = ({ itemTableData }: ItemTableProps) => {
         }`}
         columnHeaders={tableHeadings}
         // TODO: Review these values with the design team
-        columnStyles={[
-          { minWidth: 350, maxwidth: 350 },
-          { minwidth: 150, maxWidth: 200 },
-          { minwidth: 150, maxWidth: 150 },
-          { minwidth: 150, maxWidth: 150 },
-          { minwidth: 200, maxWidth: 250 },
-          { minwidth: 150, maxWidth: 200 },
-        ]}
+        columnStyles={
+          inSearchResult
+            ? [
+                { width: "33.3%", minWidth: 150, maxWidth: 250 },
+                { width: "33.3%", minWidth: 150, maxWidth: 250 },
+                { width: "33.3%", minWidth: 150, maxWidth: 250 },
+              ]
+            : [
+                { minWidth: 350, maxwidth: 350 },
+                { minwidth: 150, maxWidth: 200 },
+                { minwidth: 150, maxWidth: 150 },
+                { minwidth: 150, maxWidth: 150 },
+                { minwidth: 200, maxWidth: 250 },
+                { minwidth: 150, maxWidth: 200 },
```

Comment:
`minWidth` and `maxWidth` should use camel case. The all lowercase props are not being applied properly.